### PR TITLE
Update the sample manifests to use generic naming

### PR DESCRIPTION
**Pull request by rpothier**
### Desired Outcome

The sample manifests should have the same naming as the docs to
minimize the manual changes that are done to the manifests when updating the docs.

### Implemented Changes

Updated the naming of the manifests so they can be copied right into the docs.
Added the JWT manifests that were used in the docs. 

### Connected Issue/Story

CyberArk internal issue link: [ONYX-23699](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-23699)

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: assets/jwt-ptf-rotation.yaml [hunk 1]
```diff
@@ -19,14 +19,13 @@ spec:
         conjur.org/secrets-destination: file
         conjur.org/jwt-token-path: /var/run/secrets/tokens/jwt
         conjur.org/conjur-secrets.test-app: |
-          - test-secrets-provider-p2f-jwt-app-db/url
-          - test-secrets-provider-p2f-jwt-app-db/username
-          - test-secrets-provider-p2f-jwt-app-db/password
+          - admin-username: <path to secret in Conjur Cloud>
+          - admin-password: <path to secret in Conjur Cloud>
```
Comment (szh) [inline assets/jwt-ptf-rotation.yaml:23]: Why are there references here to Conjur Cloud?

File: assets/jwt-ptf.yaml [hunk 2]
```diff
@@ -38,10 +37,12 @@ spec:
         image: 'cyberark/secrets-provider-for-k8s:latest'
         imagePullPolicy: Always
         volumeMounts:
+        - name: conjur-status
+          mountPath: /conjur/status
         - name: podinfo
           mountPath: /conjur/podinfo
         - name: conjur-secrets
-          mountPath: /conjur/secrets
+          mountPath: /conjur/
```
Comment (szh) [inline assets/jwt-ptf.yaml:45]: Will mounting the secrets volume at `/conjur` overwrite the other volume mounts at `/conjur/status` and other subdirectories?

**Review by szh — APPROVED** [review]

File: assets/jwt-ptf-rotation.yaml (hunk 1 above)
Comment (rpothier) [reply assets/jwt-ptf-rotation.yaml:23]: This was tested in a Conjur Cloud environment.

File: assets/jwt-ptf.yaml (hunk 2 above)
Comment (rpothier) [reply assets/jwt-ptf.yaml:45]: Yea they are using /conjur and not what we were recommending. The docs are being re designed so we won't have so many of these manifests. The final one should have /conjur/secrets. Will leave this as is for now as a copy.

**Pull request merged**